common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
+        if self.isEnabledFor(AUDIT):
+            self._log(AUDIT, message, args, **kws)
+
+    logging.Logger.audit = audit
+
+
+def load_global_log_config():
+    """
+    A general method to load the overall config of the system, specifically it modifies the root handler to output
+    to sysout and sets the default log levels and format. This is expected to be called once at the start of a
+    system.
```

Comment:
Perhaps slightly clearer to say:
```suggestion
    application.
```